modules/nw-enabling-hsts-per-route.adoc

```diff
@@ -8,6 +8,13 @@
 
 HTTP strict transport security (HSTS) is implemented in the HAProxy template and applied to edge and re-encrypt routes that have the `haproxy.router.openshift.io/hsts_header` annotation.
 
+ifndef::microshift[]
+[NOTE]
+====
+If you want enable transparent end-to-end HTTP/2 connectivity in HAProxy, see "Enabling HTTP/2 Ingress connectivity" in the _Additional resources_ section.
+====
+endif::microshift[]
```

Comment:
It isn't clear to me why documentation for HSTS specifically should cross-reference the documentation for HTTP/2.

Reply:
Removed.